flink-cdc-connect/flink-cdc-pipeline-connectors/flink-cdc-pipeline-connector-starrocks/src/main/java/org/apache/flink/cdc/connectors/starrocks/sink/StarRocksDataSinkOptions.java

```diff
@@ -73,6 +73,14 @@ public class StarRocksDataSinkOptions {
                     .defaultValue(30000)
                     .withDescription("Timeout in millisecond for connecting to the `load-url`.");
 
+    public static final ConfigOption<Integer> SINK_SOCKET_TIMEOUT =
+            ConfigOptions.key("sink.socket.timeout-ms")
+                    .intType()
+                    .defaultValue(-1)
+                    .withDescription(
+                            "Supported since 1.2.10. The time duration for which the HTTP client waits for data."
```

Comment:
We should not expose the version of StarRocks sdk to user, instead, we can expose the version of FlinkCDC.

Reply:
ok